test/operance_data_row_test.dart

```diff
@@ -0,0 +1,180 @@
+// 🐦 Flutter imports:
+import 'package:flutter/material.dart';
+
+// 📦 Package imports:
+import 'package:flutter_test/flutter_test.dart';
+
+// 🌎 Project imports:
+import 'package:operance_datatable/src/operance_data_column.dart';
+import 'package:operance_datatable/src/operance_data_decoration.dart';
+import 'package:operance_datatable/src/operance_data_row.dart';
+
+void main() {
+  group('Given an OperanceDataRow instance', () {
+    final columns = <OperanceDataColumn<String>>[
+      OperanceDataColumn<String>(
+        name: 'name',
+        columnHeader: const Text('Name'),
+        cellBuilder: (context, item) => Text(item),
+      ),
+    ];
+
```

Comment:
Could maybe add a helper functions to build a test widget with Material App and Scaffold which should reduce code duplication across tests 
```suggestion
  Widget buildTestWidget(OperanceDataRow<String> dataRow) {
    return MaterialApp(
      home: Scaffold(
        body: dataRow,
      ),
    );
  }
```